crates/rover-client/schema.graphql

```diff
@@ -0,0 +1,7034 @@
+"""
```

Comment:
should this be in .gitignore? it looks like something you were using to test?

Reply:
I think until we have a strategy for automatically fetching the schema, we should keep this file checked in. This is the file that `graphql-client` uses to generate types for the queries in `rover-client`